### PR TITLE
fix: github-ci

### PR DESCRIPTION
**Pull request by zccabb**

### CODE REVIEW

File: backend/router/router.go [hunk 1]
```diff
@@ -17,4 +18,14 @@ func NewRouter(ctx context.ApplicationContext) http.Handler {
 	return e
 }
 func mountAPIs(e *gin.Engine, ctx context.ApplicationContext) {
+	api := e.Group("api")
+	{
+		api.GET("/user", user.UserGet(ctx))
+		api.GET("/ping", func(c *gin.Context) {
```
Comment (yuluo-yx) [inline backend/router/router.go:24]: This ping can be deleted and is only used in the project to test interface availability.

**Review by yuluo-yx — APPROVED** [review]
LGTM!

File: backend/router/router.go (hunk 1 above)
Comment (zccabb) [reply backend/router/router.go:24]: This is an example for the infrastructure, for the next developer to test. @oteg006

Comment (yuluo-yx) [reply backend/router/router.go:24]: > This is an example for the infrastructure, for the next developer to test. @oteg006

got it.

**Pull request merged**